<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkMITKRegionGrowingViewControls</class>
 <widget class="QWidget" name="QmitkMITKRegionGrowingViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>158</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelWarning">
     <property name="styleSheet">
      <string notr="true">QLabel { color: rgb(255, 0, 0) }</string>
     </property>
     <property name="text">
      <string>请选择一张图！</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lowerLabel">
       <property name="text">
        <string>下限:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lowerEdit">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Hu</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="upperLabel">
       <property name="text">
        <string>上限:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="upperEdit">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Hu</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="buttonPerformImageProcessing">
     <property name="toolTip">
      <string>执行区域增长分割算法</string>
     </property>
     <property name="text">
      <string>区域增长</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>201</width>
       <height>34</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
